http-clients/aws-crt-client/src/test/java/software/amazon/awssdk/http/crt/ProxyConfigurationTest.java

```diff
@@ -27,12 +27,45 @@
  */
 public class ProxyConfigurationTest {
     private static final Random RNG = new Random();
+    private static final String TEST_HOST = "foo.com";
+    private static final int TEST_PORT = 7777;
+    private static final String TEST_USER = "testuser";
+    private static final String TEST_PASSWORD = "123";
 
     @Test
     public void build_setsAllProperties() {
         verifyAllPropertiesSet(allPropertiesSetConfig());
     }
 
+    @Test
+    public void build_systemPropertyEnabled() {
+        setSystemProperty();
+        ProxyConfiguration config = ProxyConfiguration.builder().useSystemPropertyValues(true).build();
```

Comment:
Can we perhaps exclude `useSystemPropertyValues(true)` here to ensure the default value is `true`? Or have tests for both scenarios.